docs/docsite/rst/become.rst

```diff
@@ -205,6 +205,94 @@ or '/bin/chmod' as the allowed commands this will fail with ansible as those
+``````````````````
+
+Since Ansible 2.3, become can be used on Windows hosts through the ``runas``
+method. Become on Windows uses the same inventory setup and invocation
+arguments as the normal method so nothing in different from the above.
+
+While ``become`` can be used to become another user, there are other uses for
+using become with Windows hosts. One important use is to bypass some of the
+limitations that are imposed when running on WinRM such as constrained network
+delegation or forbidden system calls like the WUA API. You can use ``become``
+with the same user as ``ansible_user`` to bypass these limitations and do tasks
+that are not normally accessible in a WinRM session.
+
+.. note:: While become support was added for Windows since Ansible 2.3 it is
+    still considered experimental and can change in the future based on issues
+    and limitations that are found. Before Ansible 2.4 become would only work
+    when ``ansible_winrm_transport`` was either ``basic`` or ``credssp`` but
+    since that release it now works on all transport types.
+
+Administrative Rights
+---------------------
+
+A lot of tasks in Windows requires administrative privileges to complete.
+Ansible will attempt to run a process as an administrator and fall back to a
+limited token if that fails.
+
+Ways to become with an admin token are:
+
+* Set the ``become_user`` to the default local ``Administrator`` account. This
+  account is not any user that is a member of the local Administrators group
+  but rather the user called ``Administrator`` that is installed with Windows.
+
+* Grant the privilege ``SeTcbPrivilege`` to the user Ansible connects with on
+  WinRM. The privilege ``SeTcbPrivilege`` is a high level privilege that grants
+  full control over the operating system. No user is giving this privilege by
+  default and care should be taken if you grant a user or group to this
+  privilege. More details on this privilege and what it does can be seen
+  `here <https://technet.microsoft.com/en-us/library/dn221957(v=ws.11).aspx>`_.
+  You can use the below task to before this action on a Windows
+  host::
+
+    - name: grant the ansible user the SeTcbPrivilege right
+      win_user_right:
+        name: SeTcbPrivilege
+        users: '{{ansible_user}}'
+        action: add
+
+* Turn UAC off on the host and reboot before trying to become the user. UAC is
+  a security protocol that is designed to run accounts with the
+  ``least privilege`` principal. You can turn UAC off by running the below
+  tasks::
+
+    - name: turn UAC off
+      win_regedit:
+        path: HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\policies\system
+        name: EnableLUA
+        data: 0
+        type: dword
+        state: present
+      register: uac_result
+    
+    - name: reboot after disabling UAC
+      win_reboot:
+      when: uac_result|changed
+
+Limitations
+-----------
+
+There are currently a few known limitations with ``become`` on Windows should
```

Comment:
How about rephrasing as  'Be aware of the following limitations with ``become`` on Windows.'